### ISSUE TITLE
[KEYBOARD]: Accept Terms Prompt - SHOULD NOT have a second tab stop in the source order

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9199]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our [accept terms prompt] has two tab stops in the normal source order. This is caused by the inclusion of a second `tabIndex` attribute, which should be removed. Code sample and line number included below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Component only has one tab stop (identified by the yellow focus halo) that surrounds the entire component. Next focusable item should be the checkbox, then buttons.
- [ ] Unit tests are updated
- [ ] No axe check errors

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/accepttermsprompt/

## Possible Fixes (optional)
```diff
! AcceptTermsPrompt.jsx#L106

<div
  className="terms-scroller"
  onScroll={this.handleScroll}
- tabIndex="0">
+ >
```

## Screenshots or Trace Logs

![Screen Shot 2020-05-18 at 11 48 16 AM](https://user-images.githubusercontent.com/934879/82240469-18012780-9900-11ea-8f15-852b6d8b192a.png)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #227](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/227) via [**ZenHub**](https://www.zenhub.com/)
